domain/src/test/java/org/oppia/android/domain/classify/rules/numericexpressioninput/NumericExpressionInputMatchesUpToTrivialManipulationsRuleClassifierProviderTest.kt

```diff
@@ -76,19 +73,17 @@ class NumericExpressionInputMatchesUpToTrivialManipulationsRuleClassifierProvide
   }
 
   @Test
-  @RunParameterized(
-    Iteration("-2==-2", "answer=-2", "input=-2"),
-    Iteration("1+3.14==1+3.14", "answer=1+3.14", "input=1+3.14"),
-    Iteration(" 1 +   3.14 ==1+3.14", "answer= 1 +   3.14 ", "input=1+3.14"),
-    Iteration("1+2+3==1+2+3", "answer=1+2+3", "input=1+2+3"),
-    Iteration("1-3.14==1-3.14", "answer=1-3.14", "input=1-3.14"),
-    Iteration("2*3.14==2*3.14", "answer=2*3.14", "input=2*3.14"),
-    Iteration("2/3==2/3", "answer=2/3", "input=2/3"),
-    Iteration("2/3.14==2/3.14", "answer=2/3.14", "input=2/3.14"),
-    Iteration("2^3==2^3", "answer=2^3", "input=2^3"),
-    Iteration("2^3.14==2^3.14", "answer=2^3.14", "input=2^3.14"),
-    Iteration("sqrt(2)==sqrt(2)", "answer=sqrt(2)", "input=sqrt(2)")
-  )
+  @Iteration("-2==-2", "answer=-2", "input=-2")
+  @Iteration("1+3.14==1+3.14", "answer=1+3.14", "input=1+3.14")
+  @Iteration(" 1 +   3.14 ==1+3.14", "answer= 1 +   3.14 ", "input=1+3.14")
```

Comment:
Same question about spaces.

Reply:
Yep it's intentional--see comment above.